packages/boba/contracts/contracts/LP/L2LiquidityPool.sol

```diff
@@ -86,6 +86,9 @@ contract L2LiquidityPool is CrossDomainEnabled, ReentrancyGuardUpgradeable, Paus
     // Default gas value which can be overridden if more complex logic runs on L1.
     uint32 public DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS;
 
+    address payable private constant OVM_ETH = payable(0x4200000000000000000000000000000000000006);
```

Comment:
won't take storage slots, should be still upgradeable

Reply:
You can import it from `packages/contracts/contracts/libraries/constants/Lib_PredeployAddresses.sol`, so no need to hardcode it.